src/pages/index.js

```diff
@@ -1,12 +1,70 @@
 import React from "react"
 import { graphql } from "gatsby"
 import Layout from "../components/layout"
+import styled from "styled-components"
+import dimensions from "../style/dimensions"
+import { layoutPaddingDesktop, layoutPaddingMobile } from "../style/variables"
 
-export default function Home({ data }) {
+const HeroContainer = styled.div`
+  height: 100vh;
```

Comment:
There appears to be a bug with font scaling on Safari. Please see the demo below. Sreya encountered this same problem and fixed it – if you need advice I'm sure you can ask her! 


https://user-images.githubusercontent.com/29356116/112874021-6343a500-9090-11eb-85df-b2d48943054d.mov